src/commands/utils/target/read.ts

```diff
@@ -0,0 +1,31 @@
+import {Command, Flags} from '@oclif/core'
+import {ExecJSON} from 'inspecjs'
+import fs from 'fs'
+
+export default class ReadTarget extends Command {
+    static usage = 'target read -i <hdf-json> [-o <target-json>]'
+
+    static description = 'Read the `target` attribute in a given Heimdall Data Format JSON file and send it to stdout or write it to a file'
+
+    static examples = ['saf target read -i hdf.json -o target.json']
+
+    static flags = {
+      help: Flags.help({char: 'h'}),
+      input: Flags.string({char: 'i', required: true, description: 'An input HDF file'}),
+      output: Flags.string({char: 'o', description: 'An output `target` JSON file (otherwise the data is sent to stdout)'}),
+    }
+
+    async run() {
+      const {flags} = await this.parse(ReadTarget)
+
+      const input: ExecJSON.Execution & {target?: unknown} = JSON.parse(fs.readFileSync(flags.input, 'utf8'))
+
+      const target = input.target || {}
+
+      if (flags.output) {
+        fs.writeFileSync(flags.output, JSON.stringify(target, null, 2))
+      } else {
+        console.log(JSON.stringify(target, null, 2))
```

Comment:
A nitpick that you can determine the value of - I love to get rid of magic numbers, so making 2 a `num_spaces` or something.

Reply:
agreed

Reply:
The problem is that this constant shows up everywhere across a number of our projects.  I get that any number aside from 0 or 1 is magic and should be defined by a constant, but I feel like the '2' here is fine esp since it's never not two across the 13 usages in the saf cli repo and the ~50ish usages in the heimdall monorepo.